java/java.editor/test/unit/data/goldenfiles/org/netbeans/modules/editor/java/GoToSupportTest/javadocsnippet_LinkTag_AlongWith_HighlightTag.pass

```diff
@@ -1,10 +1,30 @@
+<<<<<<< HEAD
```

Comment:
Mmm... this `<<<<<<< HEAD` shouldn't be here, right?

Reply:
Addressed